packages/gatsby-plugin-utils/README.md

```diff
@@ -6,10 +6,38 @@
 npm install gatsby-plugin-utils
 ```
 
-### Example
+### validateOptionsSchema
+
+#### example
```

Comment:
```suggestion
The `validateOptionsSchema` function verifies that the proper data types of options were passed into a plugin from the `gatsby-config` file. It is called internally by Gatsby to validate each plugin's options when a site is started.

#### Example
```

I think a little explanation of what the function is/does could make sense before giving the example 🙂

Reply:
thanks Kyle 👍 !